### DISCUSSION TITLE
【委托】砍树的时候木头不小心卡到墙里了，还有一部分直接就卡在切换场景的边缘，拿不到，该怎么办！！

### DISCUSSION THREAD

**Discussion by MMagine** [share121/inter-knot#471] [其他]
![屏幕截图 2024-08-09 114014](https://github.com/user-attachments/assets/017abdc2-f1e3-4ced-b6d5-550f321a647e)
![屏幕截图 2024-08-09 114058](https://github.com/user-attachments/assets/9b920f1d-08c3-4fb1-8091-edd56caef4e9)
在线等，急！

**Comment by moji250** [comment]
这个好办，这个委托我接了，叫爱莲放个大进去【doge】
